formats/developers/ome-units.txt

```diff
@@ -0,0 +1,253 @@
+The OME System of Units
+=======================
+
+.. topic:: Overview
+
+   This sections explains the OME system of Units.
+
+Implementation
+^^^^^^^^^^^^^^
+
+The `ome.xsd` schema file defines new type enumerations: UnitsLength,
+UnitsTime, UnitsPressure, UnitsAngle, UnitsTemperature,
+UnitsElectricPotential, UnitsPower and UnitsFrequency. These types are used
+throughout the model. Each new unit attribute is in a pair with an existing
+attribute, e.g. PinholeSize and PinholeSizeUnit. The Xsd-Fu application then
+processes these into java classes that use types based on the new
+`ome.units.*` classes.
+
+The classes in `ome.units` are inspired by but not a direct implementation of
+the specification for a Units of Measurement API available from
+http://www.unitsofmeasurement.org.
+
+The classes have been constructed in a way that means they could in future be
+reworked as a wrapper for Units of Measurement API implementation developed
+at https://www.eclipse.org/uomo/ if future work on that project provides a
+more complete implementation.
+
+
+OME system of measurements
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Defines a system of measurements for:
+
+.. contents::
+  :depth: 1
+  :local:
+
+
+Angle
+-----
+
+Is based on the following unit:
+
+-  **rad** - radian (the SI unit of angle)
+
+And from this is derived the following units:
```

Comment:
Sorry, that was my fault! I corrected these to this and didn't spot that this is one of the few cases where 'this are' is correct.